src/Routes/WizardPage/WizardPage.tsx

```diff
@@ -47,12 +48,61 @@ const WizardPage = () => {
   const appContext = useContext(AppContext);
   const domain = appContext?.wizard.domain;
   const navigate = useNavigate();
+  const { notifySuccess, notifyWarning, notifyError, removeNotification } = useNotification();
   const [isCancelConfirmationModalOpen, SetIsCancelConfirmationModalOpen] = useState<boolean>(false);
 
   // FIXME Update the URL with the location for docs
   const linkLearnMoreAbout = 'https://access.redhat.com/articles/1586893';
   const linkLearnMoreAboutRemovingDirectoryAndDomainServices = 'https://access.redhat.com/articles/1586893';
 
+  const notifyNotCompleted = () => {
+    const notificationID = 'domain-registration-cancelled-notification';
+    notifyError({
+      id: notificationID,
+      title: 'Identity domain registration could not be completed',
+      description: (
+        <>
+          <p>You will need to re-launch the &quot;Register identity domain&quot; wizard.</p>
+          <p>
+            <Link to="/domains/wizard" onClick={() => removeNotification(notificationID)}>
+              Relaunch the wizard
+            </Link>
+          </p>
+        </>
+      ),
+    });
+  };
+
+  const notifyDomainRegistrationSuccess = () => {
+    if (!domain) return;
+
+    if (domain.auto_enrollment_enabled) {
+      notifySuccess({
+        title: 'Identity domain registration created and enabled',
+      });
+    } else {
+      notifyWarning({
+        title: 'Identity domain registration created but not enabled',
+        description: (
+          <>
+            <p>You can enable &quot;Domain auto-join on launch&quot; in the registry list.</p>
+          </>
+        ),
+      });
+    }
+  };
+
+  const notifyDomainRegistrationError = () => {
+    notifyError({
+      title: 'Issue occurred when finishing the domain registration',
+      description: (
+        <>
+          <p>Check domain in the registry list.</p>
```

Comment:
I used the message from the ticket. But I wonder if the "registry list" term is understandable.

Reply:
:+1: I think that is a good question for UXD mates.